packages/evolution-backend/src/models/migrations/20240205090500_addInfoAuditTypeToAuditTbl.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { Knex } from 'knex';
+
+const tableName = 'sv_audits';
+
+export async function up(knex: Knex): Promise<unknown> {
+    return knex.schema.alterTable(tableName, (table: Knex.TableBuilder) => {
+        table.boolean('is_info');
```

Comment:
Can an audit be both info and warning? Usually, concepts like that will use a `level` field: error, warning, info (critical, debug, trace, but we don't need those for audits I suppose)

Reply:
Merging warning and error into a level would mean long refactoring. Let's keep as is for now, and add an issue for later?